model/src/main/java/org/gluu/oxtrust/model/registration/ManagementRegReqParam.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * oxTrust is available under the MIT License (2008). See http://opensource.org/licenses/MIT for full text.
+ *
+ * Copyright (c) 2014, Gluu
+ */
+package org.gluu.oxtrust.model.registration;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * List of parameters required in {@link org.gluu.oxtrust.api.RegistrationManagementService} class.
+ *
+ * @author Shoeb Khan
+ * @version 07/08/2018
+ */
+
+@SuppressWarnings("JavadocReference")
+public enum ManagementRegReqParam {
+
+    /**
+     * Whether captcha is disabled
+     */
+    CAPTCHA_DISABLED("captchaDisabled"),
+
+    /**
+     * Selected Attributes
+     */
+    SELECTED_ATTRIBUTES("selectedAttributes");
+
+    /**
+     * Parameter name
+     */
+    private final String name;
+
+    /**
```

Comment:
It looks like this is an irrelevant comment.
We all know what are constructors, getters, setters ...
All the methods in this class are well written we don't need comments for them.
If you agree, Please remove them.